### ISSUE TITLE
SC 2.5.8 Target Size (Minimum) measurement applicability to targets not accessed through touch, mouse, pen

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#3598]
During the WCAG2ICT meeting today we were discussing the broad range of technology in regard to 2.5.8 Target Size (Minimum). Here's a link to that [discussion of target size](https://www.w3.org/2023/12/07-wcag2ict-minutes#t03). In reading the understanding doc, the research seemed to focus on touch interfaces, which are close-range targets accessed via fingertip, mouse pointer, or pen, and led to a 24x24 CSS pixel target size.

For other technologies that are less close-range, e.g. smart TVs, where one could point/click the remote to access targets, or an eye gaze for a VR technology, the viewing distance would affect the physical size per the CSS pixel calculation. Anecdotally, since your hand isn't always anchored to anything physical when pointing at your TV with a remote (to steady one's hand), the target may need to be larger to account for a reduction in accurate pointing. Has research been done to ensure the minimum target size of 24x24 CSS pixels is appropriate for those technologies? 

What would be the "normal" or "typical" viewing distance for such technologies? Should this SC be applied as written in these applications of the SC? I'm not sure there has been any research into this yet, so in the meantime it might make sense to limit the applicability of this SC to tech used in close-range (within touching distance).

**Comment by patrickhlauke** [comment]
And note this particular aspect has been discussed at length at the time when WCAG itself was being drafted, as it applies exactly the same way to different touchscreens, monitor sizes, etc. Even if we just look at computer monitors, there's a huge range of actual physical sizes of displays, which in the end can still run at the exact same CSS pixel resolution. There are simply too many variables that are outside of an author's control, so the only safe anchor is the CSS pixel measurement.

**Comment by patrickhlauke** [comment]
As an aside, when it comes to things like mouse ... it's irrelevant what the physical size and viewing distance is...what counts is by how much a mouse pointer moves on the screen. On physically larger displays, the mouse pointer moves a greater physical distance as well, as mouse pointers only move by whole pixel values (can't move the pointer to a fractional coordinate), and what counts here for target size is how accurately/confidently a user can move their pointer over a control and click/activate it ... so for this aspect, it's irrelevant how physically big / distant things are being shown

**Comment by patrickhlauke** [comment]
a much shorter, tl;dr: authors cannot detect whether the user is on a giant tv across the room using eye gaze, or a tiny touchscreen at a very high-dpi/density. and since SCs need to be applicable to content viewed/operated on any of these devices, it needed to find a specific measure that would reasonably apply as a minimum to all these situations. and it needed to be anchored on a single CSS pixel measure, because it's simply outside of the control of authors if the SC said "if this content is viewed on a large screen across the room, it must by this particular width/height, whereas if it's viewed on a smartphone, it must be this other width/height"

**Comment by samogami** [comment]
@patrickhlauke  Manufactures cannot always define px size.  With many ITC products the same models can have different suppliers or other factors can cause changes in the screen sizes and the view distance.  For different types of ITC the view distance can be different. Also, context of use and information being displayed on the screen change's "typical" view distance.  Applying this to all ITC makes a problematic SC for web applied to products with no accessibility improvements.  When view distance is unknown the css px  should not be made up with arm's length distance.

**Comment by patrickhlauke** [comment]
@samogami I am well aware of all that. which is why i was making the point that content authors are not in control of things. and neither are manufacturers.

**Comment by maryjom** [comment]
DRAFT TF RESPONSE:

The WCAG2ICT Task Force considered providing affirmative examples, but declines to make changes to add them. The reasoning is that we want our guidance to remain technology agnostic (similar to WCAG). Otherwise, it simply captures technology at the moment in time we are writing the guidance. Any time we provide examples in the document it will become stale over time. 

This is a good idea for work outside of the WCAG2ICT document which is focused on interpretation of success criteria to non-web contexts.

**Comment by patrickhlauke** [comment]
@maryjom is there any action/follow-up to be taken on this at the WCAG end?
